files/generate-vars

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/python
+try:
+    import configparser
+except ImportError:
+    import ConfigParser as configparser
+import os.path
+import shlex
+import subprocess
+import sys
+
+from subprocess import call
+
+
+class GenerateMappingFile():
+    prefix = "[Generate Mapping File] "
+    ansible_play = "../examples/dr_generate_var.yml"
```

Comment:
This won't work in RPM, becuase in RPM the examples are in:

`/usr/share/doc/ovirt-ansible-disaster-recover/examples/..`

But the path of the role is:

`/usr/share/ansible/roles/oVirt.disaster.recovery/tasks/main.yml`

Reply:
Changed to be compatible to the role